test/data/manifests/centos_8-aarch64-edge_installer-boot.json

```diff
@@ -11499,12 +11499,9 @@
       },
       "org.osbuild.ostree": {
         "items": {
-          "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff": {
+          "8119013dc0bbd4a0140d7392c3a07395399d5a12112f911bd3fc37ba6b7df6ad": {
             "remote": {
-              "url": "http://edge.example.com/repo",
-              "secrets": {
-                "name": "org.osbuild.rhsm.consumer"
-              }
```

Comment:
This is wrong.

Reply:
Fixed.  It was an issue in `gen-manifests` not setting the value when it had to.